### ISSUE TITLE
Adding a Group Initiative Removes a Non-Group Combatant from Combat

### ISSUE THREAD

**Issue by n-depinto** [fantasycalendar/FoundryVTT-MinionManager#2]
When I add a group of tokens as an initiative group, (right-clicking on the token HUD's "add to initiative" button and opening the group initiative interface) 1 Non-Group Combatant is removed from combat. I am using DND5E 2.3.1 on FoundryVTT 11.313

https://github.com/fantasycalendar/FoundryVTT-MinionManager/assets/20977202/f7e00c59-fb45-4685-b5bc-6ee78c14f5c8

**Comment by Haxxer** [comment]
Can you double check if this is not a module conflict?

**Comment by n-depinto** [comment]
Yep. I deactivated all other modules, but I am still experiencing the same behavior. Here is a vid of the same example and a screenshot of the module preferences. 

I also tried turning the debug mode on and copying the console log when doing that example, but I am not sure if I am seeing the debug messages: 

Foundry VTT | Created 3 Combatant documents with ids: [Wc2DHyUFpX38sdfe,q4jc0FAZpduvMa0l,XemJjWd1USeA6k6K] in parent Combat [DII9qXX2Tv13JCHd]
foundry.js:5744 Foundry VTT | Rendering TokenHUD
commons.js:8802 Foundry VTT | Deleted Combatant with id [Wc2DHyUFpX38sdfe] in parent Combat [DII9qXX2Tv13JCHd]
commons.js:8802 Foundry VTT | Created Combatant with id [o4t5qeQYCNt9hZ4Y] in parent Combat [DII9qXX2Tv13JCHd]
foundry.js:5744 Foundry VTT | Rendering TokenHUD
commons.js:8802 Foundry VTT | Deleted Combatant with id [q4jc0FAZpduvMa0l] in parent Combat [DII9qXX2Tv13JCHd]
commons.js:8802 Foundry VTT | Created Combatant with id [jbXo6NNJIWxhC9qA] in parent Combat [DII9qXX2Tv13JCHd]
foundry.js:5744 Foundry VTT | Rendering TokenHUD
commons.js:8802 Foundry VTT | Deleted Combatant with id [XemJjWd1USeA6k6K] in parent Combat [DII9qXX2Tv13JCHd]
commons.js:8802 Foundry VTT | Created Combatant with id [L5jtjoUrJr4MNG9K] in parent Combat [DII9qXX2Tv13JCHd]



![Screenshot 2023-10-06 155105](https://github.com/fantasycalendar/FoundryVTT-MinionManager/assets/20977202/b1f8d143-8abb-4659-a45a-a8cf457f3e2b)

https://github.com/fantasycalendar/FoundryVTT-MinionManager/assets/20977202/d3870615-5ee9-4e99-94b8-6fa53ebde3da
